app/(landing)/page.tsx

```
@@ -1,4 +1,5 @@
 import { Footer, Topbar } from '@/components/docs/layouts'
+import { Newsletter } from '@/components/docs/newsletter'
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)